src/azure-cli/azure/cli/command_modules/storage/commands.py

```diff
@@ -320,6 +326,18 @@ def get_custom_sdk(custom_module, client_factory, resource_type=ResourceType.DAT
         client_factory=file_data_service_factory,
         resource_type=ResourceType.DATA_STORAGE)
 
+    with self.command_group('storage share-rm', command_type=file_shares_mgmt_sdk,
+                            custom_command_type=get_custom_sdk('file',
+                                                               cf_mgmt_file_shares,
+                                                               resource_type=ResourceType.MGMT_STORAGE),
+                            resource_type=ResourceType.MGMT_STORAGE, min_api='2019-04-01', is_preview=True) as g:
+        g.command('create', 'create')
+        g.command('delete', 'delete')
```

Comment:
can we add confirmation for delete command?

Reply:
OK, I'll update it

---
In reply to: [368257995](https://github.com/Azure/azure-cli/pull/11649#discussion_r368257995) [](ancestors = 368257995)